lightning-types/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "lightning-types"
```

Comment:
Reading the PR description I was expecting a name like `lightning-common`, but `types` is fine too I guess